src/components/Deployment/Deployment.tsx

```diff
@@ -144,7 +145,20 @@ const Deployment = ({ onBackButtonClick }: { onBackButtonClick: () => void }) =>
           <Label>Deployed</Label>
         </Header>
         <Body>
-          {deployedChains.length == 0 && <Section>{isLoadingChainsStatus ? 'Loading...' : 'No Chains'}</Section>}
+          {deployedChains.length == 0 && (
+            <Section>
+              {isLoadingChainsStatus ? (
+                <ContentLoader viewBox="0 0 380 70">
+                  {/* Only SVG shapes */}
+                  <rect x="0" y="0" rx="5" ry="5" width="70" height="70" />
+                  <rect x="80" y="17" rx="4" ry="4" width="300" height="13" />
+                  <rect x="80" y="40" rx="3" ry="3" width="250" height="10" />
+                </ContentLoader>
```

Comment:
Since we're reusing this code, would it be better to make some custom loader components that we can use - or will the params need to change depending on what we're trying show as "loading"?